### ISSUE TITLE
Allow access to filtered records from javascript

### ISSUE THREAD

**Issue by cah-briangantzler** [miguelcobain/ember-yeti-table#276]
We have the need to gain access to the filtered records from yeti. We need access to this both outside the bounds of yeti in the hbs and also the JS. Currently they are yeilded from yeti but that only gives us access to them in the HBS, and any UI element we may attach them to has to be within the bounds of yeti table (Hmm, although given that yeilded table component now, I guess not so big an issue)

An action that could return the filtered rows would be great.

**Comment by cah-briangantzler** [comment]
This is implemented in https://github.com/miguelcobain/ember-yeti-table/pull/272

**Comment by cah-briangantzler** [comment]
Closing this as the PR is being closed.

The use case we currently was solved by the fact that yeti yields a table component (Pretty sure you did that one, hoping this isnt another case of me asking for a PR that I already did LOL). I knew the table component was there, just temporarily forgot. That being the case, I think I should always use the table component just for clarity. 

I wonder if we shouldnt change the docs for all examples to promote using the table component and deprecate the internal use (like we deprecated head and foot). This would eventually make renderTableElement no longer required, lessening the api surface.
